src/styles/BottomNavStyles.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
```

Comment:
I moved the styling from your BottomNav file to a separate file that uses `styled-components`

This will be the styling convention that we will move forward with to preserve higher readibility/organization.